### PR TITLE
Add proposal for light weighted traffic control for inference

### PR DESCRIPTION
**Pull request by ccchenjiahuan**
### Ⅰ. Describe what this PR does

Add proposal for light-weighted traffic control for inference.

### CODE REVIEW

File: docs/proposals/en/light-weighted-traffic-control-for-inference.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+
+In the AI inference scenarios, it usually involves multi-model services (loading different versions of AI models and providing inference services to users). For scenarios where multiple models coexist, it is usually necessary to do some refined routing for user inference requests. 
+
+Currently, some open source solutions such like kfserving usually rely on the heavier istio to do traffic forwarding，similar as the current implementation of kubedl.
+
+But in fact istio is a very heavy dependency, what we actually need in  inference scenario is only part of its gateway function，And if we want to do some customized serving-specific optimizations，the cost of customed maintenance for these external projects is high.
+
+So we plan to build a lightweight service-level traffic management for kubedl, not only for just simplifying the reliance of traffic management, but also for better optimize scenarios such as future model canary release , multi-stage inferencing, and batching in the future.
+
+### User case
```
Comment (jian-he) [inline docs/proposals/en/light-weighted-traffic-control-for-inference.md:13]: Use case, typo

File: docs/proposals/en/light-weighted-traffic-control-for-inference.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+current kubedl inferencing service core workflow:
+* When an inference crd created, we first sync entrypoint service to access inference service for users.
+* Sync each predictor to deploy containers mounted with specific model and also create a svc for each predictor.
+* If inference serves multiple model version simultaneously and canary policy has been set, we create virtualservice(crd in istio) regulation forwarding to different predictor svc.
+
+In the above workflow, the main point we can optimize is the dependence on virtualservice (istio). In order to reduce the external dependency and complexity, we plan to launch a customized ingress controller as a service-level gateway for each inference service.
+
+The envisaged workflow is roughly as follows:
+* When an inference crd created, We will create this inference ingress gateway as a sidecar and create a svc for this ingress gateway, it serves as the entrance of the entire inference service.
```
Comment (jian-he) [inline docs/proposals/en/light-weighted-traffic-control-for-inference.md:28]: which main container does this sidecar belongs to  ?

Comment (ccchenjiahuan) [reply docs/proposals/en/light-weighted-traffic-control-for-inference.md:28]: In my imagination, this ingress controller is bound to an inference service, so I define it as a "service sidecar", but in fact it should be a separate pod and will not coexist with any business container.

Comment (jian-he) [reply docs/proposals/en/light-weighted-traffic-control-for-inference.md:28]: cool, sidecar is a special term in Kubernetes meant for the sidecar container in a pod. maybe a better naming ?
overall looks good to me.

Looking forward to your contribution !

**Comment by SimonCqk** [comment]
lgtm, looking forward to your commits!

**Pull request merged**